modules/catalog/src/test/java/org/apache/ignite/internal/catalog/CatalogManagerRecoveryTest.java

```diff
@@ -81,8 +81,8 @@ public class CatalogManagerRecoveryTest extends BaseIgniteAbstractTest {
     private TestUpdateHandlerInterceptor interceptor;
 
     @AfterEach
-    void tearDown() throws Exception {
-        IgniteUtils.stopAll(catalogManager, metaStorageManager);
+    void tearDown() {
+        assertThat(IgniteUtils.stopAsync(catalogManager, metaStorageManager), willCompleteSuccessfully());
```

Comment:
stopAsync according to how it's implemented now is not an equivalent of an original stopAll that 've used closeAll that on its turn proceeds with components stopping even in case of exception thrown from the synchronous part of the stop.